### PR TITLE
New stuff

### PR DESCRIPTION
**Pull request by arafsheikh**

### CODE REVIEW

File: com.coala.core/src/com/coala/core/menu/BearMenu.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            ExternalUtils.runBearOnFile(file, bear);
+          } catch (IOException | InterruptedException ex) {
+            ex.printStackTrace();
+          }
+        }
+      });
+    }
+  }
+
+  private String[] getBears() {
```
Comment (arafsheikh) [inline com.coala.core/src/com/coala/core/menu/BearMenu.java]: See https://github.com/coala-analyzer/coala-eclipse/issues/16

Comment (harshdattani) [inline com.coala.core/src/com/coala/core/menu/BearMenu.java]: We are using static array, right?

Comment (arafsheikh) [inline com.coala.core/src/com/coala/core/menu/BearMenu.java]: In this case yes. But this is temporary.

File: com.coala.core/src/com/coala/core/handlers/Plugin.java [hunk 2]
```diff
@@ -32,7 +21,8 @@
   /**
    * The constructor.
    * 
-   * @throws IOException exception
+   * @throws IOException
```
Comment (Adrianzatreanu) [inline com.coala.core/src/com/coala/core/handlers/Plugin.java]: why is this on 2 lines now :D ?

File: com.coala.core/src/com/coala/core/utils/ExternalUtils.java [hunk 3]
```diff
@@ -58,7 +62,7 @@ public void onProcessComplete(int exitValue) {
 
       @Override
       public void onProcessFailed(ExecuteException executeException) {
-        System.out.println("Running coala failed.");
+        System.out.println("Running coala failed");
```
Comment (Adrianzatreanu) [inline com.coala.core/src/com/coala/core/utils/ExternalUtils.java]: i think a period here is deserved?

File: com.coala.core/src/com/coala/core/handlers/Plugin.java (hunk 2 above)
Comment (arafsheikh) [inline com.coala.core/src/com/coala/core/handlers/Plugin.java]: Eclipse formats it automatically.

File: com.coala.core/src/com/coala/core/menu/BearMenu.java (hunk 1 above)
Comment (harshdattani) [inline com.coala.core/src/com/coala/core/menu/BearMenu.java]: Got it

**Comment by harshdattani** [comment]
ack fa70d22

**Comment by harshdattani** [comment]
ack 6a1bdc9

**Comment by harshdattani** [comment]
ack 848e7ab Need one to move menu in PackageExplorer

File: com.coala.core/src/com/coala/core/handlers/Plugin.java (hunk 2 above)
Comment (harshdattani) [inline com.coala.core/src/com/coala/core/handlers/Plugin.java]: Noted!

**Comment by harshdattani** [comment]
ack 5141977

**Comment by harshdattani** [comment]
ack a622983

File: com.coala.core/src/com/coala/core/utils/ExternalUtils.java (hunk 3 above)
Comment (harshdattani) [inline com.coala.core/src/com/coala/core/utils/ExternalUtils.java]: Can add in next commit

**Comment by harshdattani** [comment]
ack 4c3befe reviewed!

**Comment by harshdattani** [comment]
ack 1d7809d

**Comment by harshdattani** [comment]
ack 1da1c83 you can merge it

**Pull request merged**